server/utils/auth.js

```diff
@@ -0,0 +1,21 @@
+const jwt = require('jsonwebtoken');
+
+// TODO move this to .ENV
+const config = {
+    secrets: {
+    // jwt: process.env.JWT_SECRET,
+    jwt: '911vmSecretPleaseChange',
```

Comment:
for now a hybrid might be `process.env.JWT_SECRET || '911vmSecretPleaseChange'` to accommodate both.